worker/runtime/backend.go

```diff
@@ -130,6 +167,64 @@ func NewGardenBackend(client libcontainerd.Client, opts ...GardenBackendOpt) (b
 		}
 	}
 
+	var hooks specs.Hooks
+	if b.ociHooksDir != "" {
+		files, err := os.ReadDir(b.ociHooksDir)
+		if err != nil {
+			return b, fmt.Errorf("ociHooksDir: %w", err)
+		}
+
+		for _, direntry := range files {
+			if direntry.IsDir() {
+				continue
+			}
+			var f = b.ociHooksDir + "/" + direntry.Name()
+			var hookJsonContent, err = os.ReadFile(f)
+			if err != nil {
+				return b, fmt.Errorf("ociHooksDir file: %w", err)
+			}
+			fmt.Println("Parsing hooks file", f)
+			var hooksParsed HookFile
+			var err2 = json.Unmarshal(hookJsonContent, &hooksParsed)
+			if err2 != nil {
+				return b, fmt.Errorf("ociHooks file failed to parse: %w", err2)
+			}
+			for _, stage := range hooksParsed.Stages {
+				fmt.Println("Add hook to stage", stage)
+				switch stage {
+				case "prestart":
+					hooks.Prestart = append(hooks.Prestart, hooksParsed.Hook)
+				case "createRuntime":
+					hooks.CreateRuntime = append(hooks.CreateRuntime, hooksParsed.Hook)
+				case "createContainer":
+					hooks.CreateContainer = append(hooks.CreateContainer, hooksParsed.Hook)
+				case "startContainer":
+					hooks.StartContainer = append(hooks.StartContainer, hooksParsed.Hook)
+				case "poststart":
+					hooks.Poststart = append(hooks.Poststart, hooksParsed.Hook)
+				case "poststop":
+					hooks.Poststop = append(hooks.Poststop, hooksParsed.Hook)
+				}
+			}
+		}
+	}
+	b.ociHooks = hooks
+
+	if b.seccompProfilePath != "" {
+		var seccompJsonContent, err = os.ReadFile(b.seccompProfilePath)
+		if err != nil {
+			return b, fmt.Errorf("seccomp file: %w", err)
+		}
+		var profile specs.LinuxSeccomp
+		var err2 = json.Unmarshal(seccompJsonContent, &profile)
+		if err2 != nil {
+			return b, fmt.Errorf("seccomp file failed to parse: %w", err2)
+		}
+		b.seccompProfile = profile
+	} else {
+		b.seccompProfile = bespec.GetDefaultSeccompProfile()
+	}
+
```

Comment:
```suggestion
	b.seccompProfile = bespec.GetDefaultSeccompProfile()
```

Reply:
I do not understand the proposed changeset.

The `if` case covers and fills `b.seccompProfile` from the provided path or fails, the `else` case should fill it with the default seccomp `b.seccompProfile`.

If the else case is removed, it will override the loaded seccomp profile with the default one.

Reply:
oh my mistake here. I thought we either want to always set it to default or the provided one. If thats not the case then we are good here.